src/smclarify/bias/metrics/common.py

```diff
@@ -175,6 +175,49 @@ def ensure_series_data_type(series: pd.Series, values: Optional[List[Any]] = Non
     raise ValueError("Data series is invalid or can't be classified as neither categorical nor continous.")
 
 
+def convert_positive_label_values(series: pd.Series, positive_label_values: List[Union[str, int, float]]) -> List:
+    """
+    Determines the type of the given data series and then do necessary type conversion to ensure the positive_lable_values
+    are of the same type as those in series.
+
+
+    Example problem when it helps:
+    The problem is that the `label_values_or_threshold` and the actual `label` values are not the same -
```

Comment:
Is this the problem? I don't think this is the problem. If this is an issue, let's simple fail. 

Let's discuss offline.

Reply:
Actually, I understand it now. Makes sense. LGTM.